.cirrus.yml

```diff
@@ -3,7 +3,8 @@ task:
   freebsd_instance:
     matrix:
       image_family: freebsd-11-4
-      image_family: freebsd-12-2
+      image_family: freebsd-12-3
+      image_family: freebsd-13-0
```

Comment:
I'm wondering if we should be running 3 versions of FreeBSD in our CI, isn't it too much?

Reply:
If you agree we can drop 11-4 which is no more maintained

Reply:
Sure, let's do that!

Reply:
done